test/e2e/quick_start_test.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var _ = Describe("Quick start", Serial, func() {
-	for _, provider := range []string{"Docker", "AWS"} {
+	for _, provider := range []string{"Docker", "AWS", "Nutanix"} {
```

Comment:
I tried running locally but it didn't work, going to add an SSH key and see whats failing
```
make e2e-test E2E_LABEL='provider:Nutanix && cni:Cilium && addonStrategy:HelmAddon'
```

From CAPI controller:
```
I0426 15:58:52.612514       1 machineset_preflight.go:140] "Performing \"Scale up\" on hold because ControlPlane quick-start-2kfshj/quick-start-havauy-tlz9c is provisioning (\"ControlPlaneIsStable\" preflight failed). The operation will continue after the preflight check(s) pass" controller="machineset" controllerGroup="cluster.x-k8s.io" controllerKind="MachineSet" MachineSet="quick-start-2kfshj/quick-start-havauy-md-0-7zwdl-w2tc5" namespace="quick-start-2kfshj" name="quick-start-havauy-md-0-7zwdl-w2tc5" reconcileID="9acdf012-5816-4e00-bf1d-ccb5d5a04a77" MachineDeployment="quick-start-2kfshj/quick-start-havauy-md-0-7zwdl" Cluster="quick-start-2kfshj/quick-start-havauy"
```

Reply:
Ahh I think I know whats happening, I was using a 1.29 image but left the k8s version at 1.28.